src/wolfgame.py

```diff
@@ -6562,6 +6563,14 @@ def relay(cli, nick, chan, rest):
     if rest.startswith("\u0001PING"):
         cli.notice(nick, rest)
         return
+    if rest.startswith("\u0001VERSION"):
+        try:
+          ans=subprocess.check_output(["git", "log", "-n", "1", "--pretty=format:%h"])
+          reply = "lykos "+str(ans.decode())+", python v"+platform.python_version()+" -- https://github.com/lykoss/lykos"
```

Comment:
use str.format instead of string concatenation, capitalize Python, drop the "v" in front of the python version (same with below in the except)